source/tutorials/getting-started-rails6.txt

```diff
@@ -14,11 +12,11 @@ Getting Started (Rails 6)
 
 .. note::
 
-  This tutorial is for Ruby on Rails 6. If this is not the version you're using choose
-  the appropriate tutorial for your Rails version from the navigation menu.
+   This tutorial is for Ruby on Rails 6. If this is not the version you're using choose
+   the appropriate tutorial for your Rails version from the navigation menu.
 
 New Application
-===============
+---------------
```

Comment:
S: Add a quick paragraph that this page covers both new and existing applications, and users should jump to whichever one they're using.